datev_export_xml/tests/test_datev_export.py

```diff
@@ -154,7 +150,7 @@ def create_out_invoice(self, customer, start_date, end_date):
                             "credit": 600.0,
                             "debit": 0.0,
                             "account_id": self.account_income.id,
-                            "analytic_account_id": self.analytic_account_it.id,
+                            "analytic_distribution": {self.analytic_account_it.id: 1},
```

Comment:
It's only a test but these are in percent which means you have 1% set for all those.